servers/rendering/renderer_rd/shaders/effects/tonemap.glsl

```diff
@@ -281,8 +294,10 @@ vec3 apply_tonemapping(vec3 color, float white) { // inputs are LINEAR, always o
 		return tonemap_reinhard(max(vec3(0.0f), color), white);
 	} else if (params.tonemapper == TONEMAPPER_FILMIC) {
 		return tonemap_filmic(max(vec3(0.0f), color), white);
-	} else { // TONEMAPPER_ACES
+	} else if (params.tonemapper == TONEMAPPER_ACES) {
 		return tonemap_aces(max(vec3(0.0f), color), white);
+	} else { // TONEMAPPER_TONY_MC_MAPFACE
```

Comment:
I would make it an explicit `else if` that checks the value.

Reply:
I amended the commit to explicitly check the enum with an `else if` in both `renderer_rd/shaders/effects/tonemap.glsl` and `gles3/shaders/tonemap_inc.glsl`.